aci-builder/bin-run/builder.go

```diff
@@ -258,6 +257,7 @@ func (b *Builder) prepareNspawnArgsAndEnv(commandPath string) ([]string, []strin
 	args = append(args, "--directory="+b.stage1Rootfs)
 	args = append(args, "--bind="+b.aciHomePath+"/:/dgr/aci-home")
 	args = append(args, "--bind="+b.aciTargetPath+"/:/dgr/aci-target")
+	args = append(args, "--bind=/etc/resolv.conf:/etc/resolv.conf")
```

Comment:
The real fix.
There is a weird things happening in stage2 that mount or do not mount it.
So explicitly mounting it should fix the issue.